hugo/content/050-tutorials/content/howto/landscape-setup/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Landscape Setup
+remote: https://github.com/gardener/landscape-setup/blob/master/README.md
```

Comment:
why isn't this page rendered in place but only a link ? Under "Contribute" the contributor guide is rendered in place. Can we also do this here for the landscape setup page ? Also the TOC could list titles contained in the linked remote page.

Reply:
The reson for the "link" is, that the collection of admin howtos not that long (n=1). This section is growing if the gardener team provides more howtos like the operation guide from vedran or the "operator of the week" guide.

In the end this is the very same layout as the "developer" section.

Reply:
the 3rd level of the TOC is autogenrated by the folder structure. This is working well for the
fixed content of the contribution guide with some pages.
This will not work for the howto and Apps because the title are much longer and 
we can have a lot more entries here. It'S just a design decision.